components/ProfilePage/FollowingTab.tsx

```diff
@@ -44,23 +46,49 @@ export const Styled = styled.div`
 export function FollowingTab({ className }: Props) {
   const { user } = useAuth()
   const uid = user?.uid
-  const subscriptionRef = collection(firestore, `/users/${uid}/subscriptions/`)
+  const subscriptionRef = collection(
+    firestore,
+    `/users/${uid}/activeTopicSubscriptions/`
+  )
+
   let billList: string[] = []
-  const [unfollowModal, setUnfollowModal] = useState<"show" | null>(null)
-  const [currentBill, setCurrentBill] = useState<string>("")
   const [billsFollowing, setBillsFollowing] = useState<string[]>([])
+  let orgsList: string[] = []
+  const [orgsFollowing, setOrgsFollowing] = useState<string[]>([])
+
+  const [currentCourt, setCurrentCourt] = useState<number>(0)
+  const [currentOrgName, setCurrentOrgName] = useState<string>("")
+  const [currentType, setCurrentType] = useState<string>("")
+  const [currentTypeId, setCurrentTypeId] = useState<string>("")
+  const [unfollowModal, setUnfollowModal] = useState<"show" | null>(null)
 
   const close = () => setUnfollowModal(null)
 
-  const handleUnfollowClick = async (billId: string) => {
-    await deleteDoc(doc(subscriptionRef, billId))
+  const handleUnfollowClick = async (
+    courtId: number,
+    type: string,
+    typeId: string
+  ) => {
+    let topicName = ""
+    if (type == "bill") {
+      topicName = "bill-".concat(courtId.toString()).concat("-").concat(typeId)
+    } else {
+      topicName = "org-".concat(typeId)
+    }
+
+    await deleteDoc(doc(subscriptionRef, topicName))
 
     setBillsFollowing([])
```

Comment:
What happens if the user is subscribed to multiple bills/orgs? will this clear out this listing?

Reply:
it clears the listing and then causes a rerender which triggers billQuery that syncs billFollowing to the current subscriptions on the backend